terraform/cloudfoundry/router_security_groups.tf

```diff
@@ -18,6 +18,7 @@ resource "aws_security_group" "web" {
     protocol  = "tcp"
     cidr_blocks = [
       "${split(",", var.web_access_cidrs)}",
```

Comment:
I think we should add office cirds here, as we always want to have access to our deployment. We should drop web access cidrs and only use the new way of adding IPs, but rename the file to e.g. additional_web_access_cidrs. For prod that would include 0.0.0.0. Or we could keep additional web access cidrs and only define additional cirds that way. It however removes ability to add comments. (but you can still comment in tfvars file).